components/organization/settings/descriptionSection.js

```diff
@@ -28,6 +29,7 @@ const DescriptionSection = ({ org }) => {
     }
     try {
       await updateOrgDescription({ organizationId: org.id, description: desc })
+      setOldDesc(desc)
```

Comment:
this was added so that if the update failed, we set the input back to it's original value